rest/src/main/java/org/apache/unomi/rest/exception/ValidationExceptionMapper.java

```diff
@@ -29,11 +29,14 @@
 @Provider
 @Component(service = ExceptionMapper.class)
 public class ValidationExceptionMapper implements ExceptionMapper<ConstraintViolationException> {
+    private static final Logger logger = LoggerFactory.getLogger(ValidationExceptionMapper.class.getName());
 
     @Override
     public Response toResponse(ConstraintViolationException exception) {
-        return Response.status(Response.Status.BAD_REQUEST).header("Content-Type", MediaType.TEXT_PLAIN)
-                .entity("Invalid received data").build();
+        exception.getConstraintViolations().forEach(constraintViolation -> logger
```

Comment:
we should propose to enable debug log level in the error message.
Then we should display the full error message in debug log, with the input value.